content/applications/websites/livechat/chatbots.rst

```diff
@@ -111,24 +117,26 @@ Phone
 *****
 
 Similar to email, this step type prompts the visitor to enter their phone number, which can be used
-at a later time to follow up with additional information, or to schedule demos, and more.
+to follow up with additional information, schedule demos, and more.
 
 .. warning::
    Due to the vast number of formats used for phone numbers worldwide, responses to this step type
-   are **not** validated for formatting.
+   are **not** validated for formatting, and can include both numbers and special characters.
 
 Forward to Operator
 *******************
 
-This step forwards the conversation to an active live chat operator, so that they can continue
-assisting the visitor. As the conversation transcript is passed on to the operator they are able to
-pick up where the chatbot left off. This not only saves time for all parties involved, it can also
-help qualify conversations before they reach live human operators.
+This step forwards the conversation to an active live chat operator, so they can continue
+assisting the visitor. As the conversation transcript is passed on to the operator, they can pick up
+where the chatbot left off. This not only saves time for all parties involved, it can also help
+qualify conversations before they reach live operators.
 
 .. note::
    If no active operator is available on the channel, the chatbot continues the conversation with
-   the visitor. Therefore, additional steps should be added after this one to ensure that there is
-   no abrupt end to the conversation.
+   the visitor. Therefore, additional steps should be added after this one to ensure there is no
+   abrupt end to the conversation. The additional steps can both inform visitors about the lack of
+   available operators (e.g.- `Uh-oh, it looks like none of our operators are available`) and
+   continue the conversation (e.g. - `Would you like to leave your email address?`).
```

Comment:
These hyphens seem unnecessary, if you leave them in please remove the space before the second one :)
```suggestion
   available operators (e.g. `Uh-oh, it looks like none of our operators are available`) and
   continue the conversation (e.g. `Would you like to leave your email address?`).
```